docs/src/appendix/cheatcodes/load.md

```diff
@@ -0,0 +1,61 @@
+# `load`
+
+> `fn load(target: ContractAddress, storage_address: felt252, size: felt252) -> Array<felt252> `
+
+Loads a `Array` of felts from `target` contract's storage, starting at `storage_address`.
```

Comment:
```suggestion
Loads `size` felts from `target` contract's storage into an `Array`, starting from `storage_address`.
```

Reply:
starting `at` is more gramatically correct
will correct though

Reply:
np, just some suggestions, adjust if needed